metrics/SystemMetrics.md

```diff
@@ -0,0 +1,35 @@
+# System Metrics
+This document serves to document the "look and feel" of the OpenCensus API for auto-collection of system metrics such as processes, memory & CPU usage, disks, etc.
+
+## Supported platforms
+TODO: Add list of supported platforms.
+
+## Common metrics collection
+These are some default metrics supported by OpenCensus system metrics package/contrib. This is to ensure consistency across language implementations, as far as practical.
+
+| Metric Name                               | Unit  | Type                       | Description                                                                                                | Labels                 |
+|-------------------------------------------|-------|----------------------------|-----------------------------------------------------------------------------------------------|-----------------|
+| system/cpu_seconds/total                  | s     | Int64 Cumulative           | Total kernel/system user CPU in seconds.                                                          | Hostname, Mode          |
+| system/processes/created                  | 1     | Int64 Cumulative           | Total number of times a process was created.                                                      | Hostname                |
+| system/processes/running                  | 1     | Int64 Gauge                | Total number of running processes.                                                                | Hostname                |
+| system/disk/total                         | By    | Int64 Gauge                | The total available system disk space in bytes.                                                   | Hostname, Volume        |
+| system/disk/used                          | By    | Int64 Gauge                | The total used system disk space in bytes.                                                        | Hostname, Volume        |
```

Comment:
add memory related stats
system/memory/used
system/memory/total
system/memory/free
all of them are Int64 Gauges.

Reply:
Sure, added.